cavityFaultDisplay.py

```diff
@@ -0,0 +1,57 @@
+from PyQt5.QtWidgets import QHBoxLayout, QLabel, QSizePolicy, QVBoxLayout
+from functools import partial
+from pydm import Display
+from typing import List
+
+from displayCavity import DISPLAY_LINAC_OBJECTS
+from Fault import Fault, PvInvalid
+
+
+class CavityFaultDisplay(Display):
+    def __init__(self, parent=None, args=None, macros=None):
+        super().__init__(parent=parent, args=args,
+                         ui_filename="frontend/cavityfaultdisplay.ui",
+                         macros=macros)
+
+        linacIdx = int(macros["linac"][1])
```

Comment:
Is Idx index?

Reply:
Yup!

Reply:
Also, what is this line? is macros a list? Or is this some fun designer shenanigan notations?

Reply:
Yeah it's the macros that got sent through designer! The json files and some extras we put in random macro fields (the things that look like `${MACRO}`)